checkov/common/vcs/vcs_schema.py

```diff
@@ -14,7 +14,7 @@ def __init__(self, schema: dict[str, Any]) -> None:
     def validate(self, data: dict[str, Any] | list[dict[str, Any]]) -> bool:
         try:
             validate(instance=data, schema=self.schema)
-        except jsonschema.exceptions.ValidationError:
-            logging.debug("validation error", exc_info=True)
+        except jsonschema.exceptions.ValidationError as e:
+            logging.debug(f"validation error {e}", exc_info=True)
```

Comment:
the `exc_info=True` argument gives the whole stack trace, is something missing or why do also add the error message again to the log message?